itests/utilities/issue_utilities/issue_tx_utility.go

```diff
@@ -22,39 +56,45 @@ func MakeTxAndGetDiffBalances[T any](suite *f.BaseSuite, testdata testdata.Issue
-	if version == 1 {
-		tx = proto.NewUnsignedIssueWithSig(testdata.Account.PublicKey, testdata.AssetName,
-			testdata.AssetDesc, testdata.Quantity, testdata.Decimals, testdata.Reissuable, testdata.Timestamp, testdata.Fee)
-	} else {
-		tx = proto.NewUnsignedIssueWithProofs(version, testdata.Account.PublicKey, testdata.AssetName, testdata.AssetDesc, testdata.Quantity, testdata.Decimals, testdata.Reissuable, nil, testdata.Timestamp, testdata.Fee)
-	}
-	err := tx.Sign(testdata.ChainID, testdata.Account.SecretKey)
-	txJson := utl.GetTransactionJsonOrErrMsg(tx)
-	suite.T().Logf("Issue Transaction JSON after sign: %s", txJson)
-	require.NoError(suite.T(), err, "failed to create proofs from signature")
-	return tx
+
+func NewSignIssueTransactionWithTestData[T any](suite *f.BaseSuite, version byte, testdata testdata.IssueTestData[T]) proto.Transaction {
+	return NewSignIssueTransaction(suite, version, testdata.ChainID, testdata.Account.PublicKey, testdata.Account.SecretKey,
+		testdata.AssetName, testdata.AssetDesc, testdata.Quantity, testdata.Timestamp, testdata.Fee, testdata.Decimals,
+		testdata.Reissuable)
 }
 
-func IssueSend[T any](suite *f.BaseSuite, testdata testdata.IssueTestData[T], version byte,
+func IssueSendWithTestData[T any](suite *f.BaseSuite, testdata testdata.IssueTestData[T], version byte,
 	waitForTx bool) utl.ConsideredTransaction {
-	tx := NewSignIssueTransaction(suite, version, testdata)
+	tx := NewSignIssueTransactionWithTestData(suite, version, testdata)
 	return utl.SendAndWaitTransaction(suite, tx, testdata.ChainID, waitForTx)
 }
 
-func IssueBroadcast[T any](suite *f.BaseSuite, testdata testdata.IssueTestData[T], version byte,
+func IssueBroadcastWithTestData[T any](suite *f.BaseSuite, testdata testdata.IssueTestData[T], version byte,
 	waitForTx bool) utl.ConsideredTransaction {
-	tx := NewSignIssueTransaction(suite, version, testdata)
+	tx := NewSignIssueTransactionWithTestData(suite, version, testdata)
 	return utl.BroadcastAndWaitTransaction(suite, tx, testdata.ChainID, waitForTx)
 }
 
 func SendIssueTxAndGetWavesBalances[T any](suite *f.BaseSuite, testdata testdata.IssueTestData[T], version byte,
 	waitForTx bool) (utl.ConsideredTransaction, utl.BalanceInWaves, utl.BalanceInWaves) {
-	return MakeTxAndGetDiffBalances(suite, testdata, version, waitForTx, IssueSend[T])
+	return MakeTxAndGetDiffBalances(suite, testdata, version, waitForTx, IssueSendWithTestData[T])
 }
 
 func BroadcastIssueTxAndGetWavesBalances[T any](suite *f.BaseSuite, testdata testdata.IssueTestData[T], version byte,
 	waitForTx bool) (utl.ConsideredTransaction, utl.BalanceInWaves, utl.BalanceInWaves) {
-	return MakeTxAndGetDiffBalances(suite, testdata, version, waitForTx, IssueBroadcast[T])
+	return MakeTxAndGetDiffBalances(suite, testdata, version, waitForTx, IssueBroadcastWithTestData[T])
+}
+
+func IssueAssetAmount(suite *f.BaseSuite, version byte, scheme proto.Scheme, accountNumber int,
+	assetAmount ...uint64) crypto.Digest {
```

Comment:
Why do you use `...` here? I can't find any usages where `len(assetAmount) != 1`.

Reply:
Fix.